src/go/privileged-service/pkg/port/server.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright © 2021 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package port
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net"
+
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/svc/debug"
+)
+
+const network = "tcp"
+
+// Server is a port server listening for port events over vtunnel
+type Server struct {
+	host        string
+	port        int
+	eventLogger debug.Log
+	quit        chan interface{}
+	listener    net.Listener
+	stopped     bool
+}
+
+func NewServer(addr string, port int, elog debug.Log) *Server {
+	return &Server{
+		host:        addr,
+		port:        port,
+		eventLogger: elog,
+		stopped:     true,
+	}
+}
+
+// Start initiates the port server on a given host:port
+func (s *Server) Start(errCh chan error) {
```

Comment:
Should clarify that `errCh` is only written to during initial startup (not once things got under way); how supervisor uses this is confusing otherwise.